src/api/BioLink.js

```diff
@@ -296,6 +296,17 @@ export async function getSources() {
   const params = new URLSearchParams();
 
   const bioentityResp = await axios.get(url, { params });
+
+  // remove this after we get this stuff from the API
+  for (let i = 0; i < bioentityResp.data.length; i++) {
+    let sourceDisplayName = bioentityResp.data[i].id.split(/[:.]+/)[1];
+    sourceDisplayName = sourceDisplayName.charAt(0).toUpperCase() + sourceDisplayName.slice(1);
+    bioentityResp.data[i].sourceDisplayName = sourceDisplayName;
+    bioentityResp.data[i].ttlUrl = 'https://data.monarchinitiative.org/ttl/' + bioentityResp.data[i].id.split(':')[1];
+    bioentityResp.data[i].sourceVersion = bioentityResp.data[i].meta.version[0];
+    bioentityResp.data[i].monarchDataReleaseDate = '2019-02-22';
```

Comment:
@justaddcoffee We do get release date from the API. 

https://api.monarchinitiative.org/api/metadata/datasets

For now we can treat `version` field of the response as Release Date.

I emphasize because there are datasets that don't have the same `2019-02-22` release date. For Example: `MonarchData:udp.ttl`

Reply:
Okay, thanks @deepakunni3. Can you clarify this a bit? 

I was thinking though that we need to display at least these things here:
1) the version according to the source (e.g. we ingested the data that MGI released on 2018-11-01), what I am calling `sourceVersion`
and 
2) the date when we released our data (here I think that is 2019-02-22) - what I am calling `monarchDataReleaseDate`
3) (maybe) the date in which dipper ingested the data - Dipper doesn't explicitly record this currently IIUC, so I am not trying to display it here

I assume that `meta.version[0]` is 1), is that right? And this is separate information from 2) the date in which we made our data release?

Reply:
Odd. So far it looks like both `sourceVersion` and release date are the same (as provided by SciGraph).

You are right. Ideally, we should be able to differentiate between Monarch release date and Dipper ingest date. 
While we don't have this, perhaps `sourceVersion` and `monarchDataReleaseDate` can be the same for now.

Reply:
Okay, I'll leave it as is for now, or feel free to alter the above code